### PR TITLE
Update mainbot.py

### PR DESCRIPTION
**Pull request by p-papnai**

### CODE REVIEW

File: mainbot.py [hunk 1]
```diff
@@ -1 +1,41 @@
...
+            'no doubt no doubt no doubt no doubt.'
+        ),
+    ]
+
+    if message.content == '99!':
+        response = random.choice(brooklyn_99_quotes)
+        await message.channel.send(response)
+
+
+client.run('NzU4OTk4MDI2NzkzNjQ4MTQw.X23F_w.s9jPxSLnCdzbXcjw6z-UZhSLiYQ')
```
Comment (ayushsingh0707) [inline mainbot.py:41]: remove the token from here

File: mainbot.py [hunk 2]
```diff
@@ -1 +1,41 @@
...
+
+
+    brooklyn_99_quotes = [
+        'I\'m the human form of the 💯 emoji.',
+        'Bingpot!',
+        (
+            'Cool. Cool cool cool cool cool cool cool, '
+            'no doubt no doubt no doubt no doubt.'
+        ),
+    ]
```
Comment (ayushsingh0707) [inline mainbot.py:34]: is it running?

File: mainbot.py [hunk 3]
```diff
@@ -1 +1,41 @@
...
+
+
+
+import discord
+from dotenv import load_dotenv
+
+
+load_dotenv()
+'''TOKEN = os.getenv(DISCORD_TOKEN)'''
+
```
Comment (ayushsingh0707) [inline mainbot.py:15]: use this for giving token to client.run()

**Review by ayushsingh0707 — CHANGES_REQUESTED** [review]

File: mainbot.py (hunk 3 above)
Comment (p-papnai) [reply mainbot.py:15]: ok dear

**Comment by ayushsingh0707** [comment]
issues resolved in another pull request by p_panai

**Pull request closed without merging**